qiita_pet/handlers/study_handlers/edit_handlers.py

```diff
@@ -264,9 +264,11 @@ def post(self, study=None):
                    'successfully created' %
                    (the_study.id, form_data.data['study_title'][0]))
 
-        # Add the environmental packages
-        the_study.environmental_packages = form_data.data[
-            'environmental_packages']
+        # Add the environmental packages, this attribute can only be edited
+        # if the study is public, otherwise this cannot be changed
```

Comment:
i.e., I think this should only be editable if the study is sandboxed (even being private would preclude changing this field)